contracts/PvpArena.sol

```diff
@@ -20,13 +22,17 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
     struct Fighter {
         uint256 characterID;
         uint256 weaponID;
+        uint256 shieldID;
         uint256 wager;
+        bool useShield;
```

Comment:
For data structures it is usually preferable to use declarative rather than imperative names 
```suggestion
        bool usingShield
```